data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,488 @@ mission "Unfettered returning home"
+				to display
+					not "Hai Legacy 3: offered"
+			`	"We will give you another ship so you can leave, but you must surrender this one," one of them explains to you in a menacing tone.`
+			`	Another one steps forward, "We will allow to keep the outfits, provided you give it to us on Darkcloak," they say in a somewhat diplomatic tone.`
+				to display
+					not "flagship planet: Darkcloak"
+			`	Another one steps forward, "Since you so kindly came to Darkcloak to deliver the ship to us, we will allow you to buy back any outfits you may still desire, as a courtesy," they say in a somewhat diplomatic tone.`
+				to display
+					has "flagship planet: Darkcloak"
+			choice
+				`	(Let them have it.)`
+				`	(Take off, now!)`
+					depart
+			`	You and your crew exit what was briefly your ship, and you inform the local Unfettered militia that it is now theirs. The Unfettered that were assembling around your ship happily disperse, except for one, probably the leader judging by their rare dual pulse pistols. They come to talk to you, weapons in hand but lowered. "We are happy you could see reason and decided to hand over the ship. In exchange, we have decided to provide you with a suitable replacement, that should hopefully allow you to exit our space safely.`
+			`	"That relic was not meant for you, but remember that there are other ways for you to prove yourself." The leader of these Unfettered continues with blatantly forced casualness. "Say, for example, bringing us a Jump Drive... or two... that for sure will be earning you favor, and prove you can fight.`
+				to display
+					not "Unfettered Jump Drive 1: offered"
+			`	"For now, you have enough to worry about with all of the missiles that will be heading your way. If I were you I'd install a lot of jammers. You'll need them, ha!" A general laughter erupts, and they let you get comfortable in the new ship. You may want to get a better one, or at least upgrade this one in order to escape safely. It may be wise to bribes to get out in one piece.`
+			action
+				give ship "Lightning Bug (Unfettered Shipyards)" "Good Will"
+				take ship "Anomalocaris"
+					id "hai legacy"
+
+mission "Unfettered Landing Permit Lost"
+	landing
+	invisible
+	source
+		attributes "unfettered"
+	to offer
+		has "Hai Legacy 3: done"
+		not "outfit (all): Unfettered Landing Permit"
+		not "Unfettered Jump Drive 1: offered"
+		"reputation: Hai (Unfettered)" >= 0
+	on offer
+		conversation
+			"As you land, the Unfettered tell you they have learned of the loss of the Anomalocaris, thereby ending your landing access and friendly relations with them. They also, very unsubtly, suggest you find and bring them a Jump Drive to atone for your carelessness."
+		log "Lost access to Unfettered space when they learned about the loss of the Anomalocaris."
+		"reputation: Hai (Unfettered)" <?= -500
```

Comment:
Since the Unfettered care so much about the ship in the mission, it makes sense that this still occurs regardless of the JD mission, which you still could do to get their favor back, so remove the to offerr condition checking that.

Instead of just saying the Unfettered learned about it, change up the conversation. Have a group of Unfettered or someone approach you as you land, asking what happened to the Anomalocaris (though they know at this point that you lost it somehow, due to a broken signal or something). Then, give the player some choices: 

They're sorry, or lost it in a fight, but the Unfettered blames the player's incompetence.
The player asks how they know, and they say they lost the signal from the AI or something.
The player won't tell them.

Of course, all three lead to the rep drop anyway.